lib/rubocop/cop/style/arguments_forwarding.rb

```diff
@@ -356,6 +357,12 @@ def use_anonymous_forwarding?
           cop_config.fetch('UseAnonymousForwarding', false)
         end
 
+        def sent_inside_block?(send_classifications)
```

Comment:
It seems to refer to `send`, so how about `send_inside_block?`, or if it refers to a method call in general, what do you think about names like `called_inside_block?` or `invoked_inside_block?` (or others...)?

Reply:
Sure thing, updated!